clang/lib/Basic/OpenMPKinds.cpp

```diff
@@ -665,60 +619,44 @@ bool clang::isOpenMPTargetDataManagementDirective(OpenMPDirectiveKind DKind) {
 }
 
 bool clang::isOpenMPNestingTeamsDirective(OpenMPDirectiveKind DKind) {
-  return DKind == OMPD_teams || DKind == OMPD_teams_distribute ||
-         DKind == OMPD_teams_distribute_simd ||
-         DKind == OMPD_teams_distribute_parallel_for_simd ||
-         DKind == OMPD_teams_distribute_parallel_for ||
-         DKind == OMPD_teams_loop;
+  if (DKind == OMPD_teams)
+    return true;
+  auto leafs = getLeafConstructs(DKind);
```

Comment:
Leafs

Reply:
Done

Reply:
Plural of 'leaf' is 'leaves'.  'leafs' is a canadian hockey team :D

Reply:
Leafs is an accepted form.  I used to live in Toronto, btw.  :)

Reply:
> Leafs is an accepted form.

Could you point to some source where that's described? After a quick search, I find that "leaves" is the widely accepted plural of "leaf", and that "leafs" is only recognized by certain dictionaries. One instance where it's not: https://dictionary.cambridge.org/dictionary/english/leaf.

Reply:
Merriam-Webster is more oriented towards American English.  The [entry for leaf](https://www.merriam-webster.com/dictionary/leaf) shows "leafs" as an alternative plural form.

Reply:
Huh, interestingly, that Merriam Webster link is the only one I can find that acknowledges `leafs` as a plural for the general case.  The only other case I could find was wiktionary: https://en.wiktionary.org/wiki/leaf

Which shows that `leafs` is valid in 2 cases: when talking about multi-section doors, or when used as a pejorative for a Canadian.  So I don't really know anything anymore.

That said, while LLVM's policy is only 'english' and doesn't specify American, Canadian, or British English, by practice we use American English spellings for things.

Reply:
If you strongly prefer the more conformist spelling, I can change it...  :)

Reply:
'strongly prefer'?  No.  Mildly? Yeah, mildly.  Perhaps slightly more than mildly because it is in an API and I'd prefer we be consistent with the 'more common' spelling.

Reply:
Uh, the "leafs" is only used as a local variable name, the function name has singular "Leaf".

Reply:
I've changed it.